MetaMask/metamask-extension
# UX: Added drag and drop to network list

This PR is to add Drag and Drop Support to Networks List

## **Manual testing steps**

1. Run MetaMask with `NETWORK_ACCOUNT_DND` feature flag
2. Open the NetworkListMenu
3. Drag and Drop the Items inside the menu
4. Close extension,check the list stays updated
5. Add a new network, check the added network is added at the bottom of the list and the remaning order stays same
6. Delete a new network, check the deleted network is removed from the list and the remaning order stays sane

## **Screenshots/Recordings**


### **After**


https://github.com/MetaMask/metamask-extension/assets/39872794/c7012122-3301-49a8-adf3-e64d2a609c49





## **Related issues**

_Fixes #20919 

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
